### PR TITLE
Life parameteres

### PR DESCRIPTION
**Pull request by lkszjakubowski**

### CODE REVIEW

File: js/game.js [hunk 1]
```diff
@@ -1,12 +1,79 @@
...
-    console.log("Game started");
+    this.tamagotchi.mount({
+      healthElement,
+      hungerElement,
+      energyElement,
+      funElement,
+    });
+  };
+
+  decreaseStats = ({
```
Comment (Sumick) [inline js/game.js:17]: **nitpick:** This method is definitely too big. Let's split it in smaller, reusable and domain specific code pieces. Example: `decreaseHealth() {...}, decreaseEnergy() {...}` etc. You should be able to call such methods independently

File: js/game.js [hunk 2]
```diff
@@ -1,12 +1,79 @@
...
+    healthElement,
+    hungerElement,
+    energyElement,
+    funElement,
+  }) => {
+    const interval = setInterval(() => {
```
Comment (Sumick) [inline js/game.js:23]: **nitpick:** Your intervals are closed in this specific method scope. What if you'd like to stop specific interval by calling any action that is not a part of that `decreaseStats` method? You don't have any control right now. Try to mount such intervals in the constructor. Eg. `this.energyDecreaseInterval = setInterval(() => {...}, 1000)`. By doing that you can access such interval from any part of your class scope, giving you full control

**Review by Sumick — CHANGES_REQUESTED** [review]
Nice job - it works properly, but we need to focus on getting this code more maintainable.

- [x] Split this huge `decreaseStats` method into smaller, reusable pieces
- [x] Fix intervals, so they are accessible within whole class

File: js/game.js (hunk 1 above)
Comment (lkszjakubowski) [reply js/game.js]: thanks for the tips! updated my code

Comment (lkszjakubowski) [reply js/game.js]: just noticed a weird behavior of the health paremater, interval skips values

**Review by Sumick — APPROVED** [review]
Amazing job! It looks great 👌 

PS. Next time, please re-send review request from Nerdbord task - without it don't have any notification that you are ready to review, thanks:)

**Pull request merged**